torchvision/transforms/v2/functional/_meta.py

```diff
@@ -345,30 +348,22 @@ def convert_dtype_image_tensor(image: torch.Tensor, dtype: torch.dtype = torch.f
             return image.to(dtype).bitwise_left_shift_(num_value_bits_output - num_value_bits_input)
 
 
-# We changed the name to align it with the new naming scheme. Still, `convert_image_dtype` is
-# prevalent and well understood. Thus, we just alias it without deprecating the old name.
-convert_image_dtype = convert_dtype_image_tensor
```

Comment:
We still need to keep this alias for BC, right? Even more so, we need

```python
def convert_image_dtype(image: torch.Tensor, dtype: torch.dtype=torch.float32) -> torch.Tensor:
    return to_dtype_image_tensor(image, dtype=dtype, scale=True)
```

since I don't think JIT supports `functools.partial`.

Reply:
ah yeah, I didn't realize we had this in V1. Thanks